### PR TITLE
MODBULKOPS-204 - Item record's column names cleanup 

### PR DESCRIPTION
**Pull request by AntonAntonich**
[MODBULKOPS-204](https://folio-org.atlassian.net/browse/MODBULKOPS-204)

## Purpose
 
Currently the list of column names for item records is ordered almost randomly and it includes properties that are important for backend processes but are normally not exposed in the inventory UI.
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/bulkops/domain/bean/Item.java [hunk 1]
```diff
@@ -1,374 +1,313 @@
...
 import com.opencsv.bean.CsvCustomBindByPosition;
-
 import jakarta.validation.Valid;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 import lombok.With;
+import org.folio.bulkops.domain.converter.*;
```
Comment (alekGbuz) [inline src/main/java/org/folio/bulkops/domain/bean/Item.java:15]: @AntonAntonich please use import for all needed classes.

File: src/test/java/org/folio/bulkops/TestEntity.java [hunk 2]
```diff
@@ -85,11 +84,11 @@ public BulkOperationsEntity entity() {
         .numberOfMissingPieces("1")
         .missingPieces("missing")
         .missingPiecesDate("01-01-2000")
-        .itemDamagedStatusId("damaged-status-id")
+        .itemDamagedStatus("damaged-status-id")
         .itemDamagedStatusDate("01-2001")
         .administrativeNotes(List.of("administrative-note-1", "administrative-note-2"))
         .notes(List.of(ItemNote.builder().build()))
-        .circulationNotes(List.of(CirculationNote.builder().build()))
+//        .circulationNotes(List.of(CirculationNote.builder().build()))
```
Comment (alekGbuz) [inline src/test/java/org/folio/bulkops/TestEntity.java:91]: @AntonAntonich please uncomment the code logic

File: src/test/java/org/folio/bulkops/controller/BulkOperationControllerTest.java [hunk 3]
```diff
@@ -23,12 +32,12 @@
...
 import java.util.stream.IntStream;
+import java.util.stream.Stream;
 
 import lombok.SneakyThrows;
 import org.folio.bulkops.BaseTest;
 import org.folio.bulkops.client.RemoteFileSystemClient;
-import org.folio.bulkops.domain.bean.Personal;
-import org.folio.bulkops.domain.bean.User;
+import org.folio.bulkops.domain.bean.*;
 import org.folio.bulkops.domain.dto.FileContentType;
```
Comment (alekGbuz) [inline src/test/java/org/folio/bulkops/controller/BulkOperationControllerTest.java:41]: @AntonAntonich please fix import

File: src/test/java/org/folio/bulkops/service/OpenCSVConverterTest.java [hunk 4]
```diff
@@ -8,58 +8,7 @@
...
-import org.folio.bulkops.domain.bean.SelectField;
-import org.folio.bulkops.domain.bean.SelectFieldOption;
-import org.folio.bulkops.domain.bean.SelectFieldOptions;
-import org.folio.bulkops.domain.bean.StatisticalCode;
-import org.folio.bulkops.domain.bean.StatisticalCodeCollection;
-import org.folio.bulkops.domain.bean.TextField;
-import org.folio.bulkops.domain.bean.User;
-import org.folio.bulkops.domain.bean.UserGroup;
-import org.folio.bulkops.domain.bean.UserGroupCollection;
+import org.folio.bulkops.domain.bean.*;
```
Comment (alekGbuz) [inline src/test/java/org/folio/bulkops/service/OpenCSVConverterTest.java:11]: @AntonAntonich it is necessary to import all needed classes

File: src/main/java/org/folio/bulkops/domain/bean/Item.java (hunk 1 above)
Comment (AntonAntonich) [reply src/main/java/org/folio/bulkops/domain/bean/Item.java]: done

File: src/test/java/org/folio/bulkops/TestEntity.java (hunk 2 above)
Comment (AntonAntonich) [reply src/test/java/org/folio/bulkops/TestEntity.java]: done

File: src/test/java/org/folio/bulkops/controller/BulkOperationControllerTest.java (hunk 3 above)
Comment (AntonAntonich) [reply src/test/java/org/folio/bulkops/controller/BulkOperationControllerTest.java:46]: done

File: src/test/java/org/folio/bulkops/service/OpenCSVConverterTest.java (hunk 4 above)
Comment (AntonAntonich) [reply src/test/java/org/folio/bulkops/service/OpenCSVConverterTest.java]: done

**Review by alekGbuz — APPROVED** [review]

**Pull request merged**